code/game/machinery/computer/cargo.dm

```diff
@@ -155,7 +155,7 @@ For vending packs, see vending_packs.dm*/
 		if(3)
 			result = pin_query(user)
 	if(!result) //This saves a lot of pasted to_chat everywhere else
-		if(can_order_contraband)
+		if(can_order_contraband || hacked)
```

Comment:
Unatomic change

Reply:
Redundancy, contraband access will already remove access restrictions

Reply:
If it's redundant, remove it
The way I read it, it makes it so hacking it - with iron - will yield the same effect than emaging

Did you test it

Reply:
boy2man is correct in that it is just redundant if this change goes through
Shifty is correct in that there's no reason to have a redundant || statement

Reply:
This is the code for checking permissions whether you can actually order anything with the console with your level of access, can_order_contraband comes from using the soldering iron on the console board, hacked comes from emagging, and I don't feel like moving a single finger to remove the redundancy unless I'm directly told it results in a bad thing. The purpose of the PR is that emagging also unlocks contraband, what's written is written. You've switched places with me are now the one who claims stuff without informing themselves.

Reply:
Right.
Then remove the `||` part

Reply:
To be clear, this is a style issue

And style issues are important

Reply:
Why is this particular case important? The code is never going to devolve into oldcoder vomit

Reply:
If the effects of can_order_contraband or hacked change, it would be unintuitive and they'll be calling *you* the oldcoder.

Listen, there's no reason to set up a tripwire issue for someone further down the line. Just take it out. You would have spent less time if you just did it, you can even do it from the web editor. At this point you're only arguing for the sake of wanting to be excepted from style review, which will inevitably not succeed because even I have to kowtow to style demands.

Reply:
Nah bro, I just wanted to make collabs consider if this flavor of autism regarding changes that don't actually affect anything is really worth it but I see nothing's gonna change so w/e
>tripwire issue

It hasn't changed in the last few years and it's likely it's not gonna change years later, what oldcoders did were stupid fundamental stuff and nobody including me has any right to compare others to oldcoders. Ping me when someone in the future that doesn't even know about this conversation adds a different flavor of "ignoring access" separate from contraband, and then I'll say you're right.